libs/apnos/README.md

```diff
@@ -1 +1,4 @@
 ## AP NOS Library
+###apnos.py : This Library Consists of the following-
+    1. class APNOS : Library to SSH and Pull the information from AP
```

Comment:
I thought we had SSH support and serial support. We should mention that.

Reply:
I can update the comments in next commit